core/build.sbt

```diff
@@ -274,6 +274,8 @@ lazy val blueprint =
       scalafmtOnCompile := true,
       libraryDependencies ++= Vector(
             SprayJson,
+            Skuber,
```

Comment:
Skuber and Jackson can now be removed, right?

Reply:
done ..